met-api/tests/unit/services/test_submission.py

```diff
@@ -50,6 +54,38 @@ def test_create_submission(session):  # pylint:disable=unused-argument
     assert actual_email_verification['is_active'] is False
 
 
+def test_update_submission(session):  # pylint:disable=unused-argument
+    """Assert that a submission can be updated using update_comments."""
+    survey, eng = factory_survey_and_eng_model()
+    email_verification = factory_email_verification(survey.id)
+    participant = factory_participant_model()
+    factory_engagement_setting_model(eng.id)
+    submission_request: SubmissionSchema = {
+        'submission_json': {"test_question": "test answer"},
+        'engagement_id': eng.id,
+        'survey_id': survey.id,
+        'participant_id': participant.id,
+        'verification_token': email_verification.verification_token,
+    }
+    submission = SubmissionService().create(
+        email_verification.verification_token, submission_request)
+    # pretend the comment was rejected
+    submission.comment_status_id = Status.Rejected.value
```

Comment:
Maybe this isn't a big deal but is there a way to get the service to set the value to `comment_status_id` rather than manually setting it? How much are we really testing the service if we're setting the value?